src/main/java/org/spongepowered/common/mixin/core/entity/player/MixinEntityPlayerMP.java

```diff
@@ -245,6 +228,10 @@ public PlayerConnection getConnection() {
         return (PlayerConnection) this.playerNetServerHandler;
     }
 
+    public void setBedLocation(@Nullable Location location) {
+        this.spawnChunk = location != null ? VecHelper.toBlockPos(location.getPosition()) : null;
```

Comment:
Due to the fact that it's not in the AT anymore (but it would need to be) I'll remove it.